src/Simplex/Chat.hs

```diff
@@ -258,14 +258,26 @@ processChatCommand user@User {userId, profile} = \case
     when (userRole < GRAdmin || userRole < memRole) $ chatError CEGroupUserRole
     when (memberStatus membership == GSMemInvited) $ chatError (CEGroupNotJoined gName)
     unless (memberActive membership) $ chatError CEGroupMemberNotActive
-    when (isJust $ contactMember contact members) $ chatError (CEGroupDuplicateMember cName)
-    gVar <- asks idsDrg
-    (agentConnId, cReq) <- withAgent (`createConnection` SCMInvitation)
-    GroupMember {memberId} <- withStore $ \st -> createContactGroupMember st gVar user groupId contact memRole agentConnId
-    let msg = XGrpInv $ GroupInvitation (userMemberId, userRole) (memberId, memRole) cReq groupProfile
-    sendDirectMessage (contactConn contact) msg
-    showSentGroupInvitation gName cName
-    setActive $ ActiveG gName
+    case contactMember contact members of
+      Nothing -> do
+        gVar <- asks idsDrg
+        (agentConnId, cReq) <- withAgent (`createConnection` SCMInvitation)
+        GroupMember {memberId} <- withStore $ \st -> createContactGroupMemberWithInvitation st gVar user groupId contact memRole agentConnId cReq
+        sendInvitation contact userMemberId userRole memberId groupProfile cReq
+      Just GroupMember {groupMemberId, memberId, memberStatus} -> do
+        if memberStatus == GSMemInvited
+          then do
+            contactRequest <- withStore $ \st -> getContactGroupMemberInvitation st user groupMemberId
+            case contactRequest of
+              Nothing -> showCannotResendInvitation gName cName
+              Just cReq -> sendInvitation contact userMemberId userRole memberId groupProfile cReq
+          else chatError (CEGroupDuplicateMember cName)
```

Comment:
```suggestion
      Just GroupMember {groupMemberId, memberId, memberStatus}
        | memberStatus == GSMemInvited -> do
          withStore $ \st -> getContactGroupMemberInvitation st user groupMemberId >>= \case
            Just cReq -> sendInvitation contact userMemberId userRole memberId groupProfile cReq
            _ -> showCannotResendInvitation gName cName
        | otherwise -> chatError (CEGroupDuplicateMember cName)
```